tests/model_serving/model_runtime/mlserver/conftest.py

```
@@ -0,0 +1,158 @@
+        namespace=py_config["applications_namespace"],
+    ) as tp:
+        yield tp
+
+
+@pytest.fixture(scope="class")
+def protocol(request: FixtureRequest) -> str:
+    return request.param["protocol_type"]
+
+
+@pytest.fixture(scope="class")
+def serving_runtime(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)